src/validation.cpp

```diff
@@ -4824,6 +4824,28 @@ std::vector<Chainstate*> ChainstateManager::GetAll()
     return out;
 }
 
+std::vector<Chainstate*> ChainstateManager::GetAllForBlockDownload()
+{
+    AssertLockHeld(::cs_main);
+    std::vector<Chainstate*> out;
+
+    bool snapshot_in_ibd{
+        m_snapshot_chainstate && m_snapshot_chainstate->IsInitialBlockDownload()};
+
+    if (m_snapshot_chainstate) {
```

Comment:
In commit "validation: add ChainstateManager::GetAllForBlockDownload()" (42f5b6b03bba1ef1c1bc6c225606d119c4e27169)

After #24008 will this need to change to `m_snapshot_chainstate && IsUsable(m_snapshot_chainstate)`?